ducktape/command_line/main.py

```diff
@@ -108,8 +107,12 @@ def main():
         print test_classes
         sys.exit(0)
 
+    # Initializing the cluster is slow, so do so only if
```

Comment:
Oh, this makes so much sense! I wondered why test discovery would always take so long.